homestar-runtime/src/runner/response.rs

```diff
@@ -78,7 +78,18 @@ impl show::ConsoleTable for AckWorkflow {
         resource_table
             .with(Modify::new(Rows::first()).with(Format::content(|_s| "Resources".to_string())));
 
-        let tbl = col![table, resource_table].default();
+        let mut progress_table = Table::new(
+            self.progress
+                .iter()
+                .map(|v| v.to_string())
+                .collect::<Vec<String>>(),
+        );
+
+        progress_table.with(
+            Modify::new(Rows::first()).with(Format::content(|_s| "Receipts Computed".to_string())),
```

Comment:
@bgins I think we show tasks CIDs for a workflow. Maybe this table can correlate tasks to receipts as well? Then, do you know which tasks are completed?

Reply:
After chatting with @bgins, the current piece we'll do is showcase the ran (invocation cid) and instruction cid (for runtime receipts). We'll do correlation with workflow static info in a separate PR, tracked in #489.